CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.1] - 2024-11-26
+-   Update dokka and github actions to compile correctly
```

Comment:
You'll want to move this comment up to the Unreleased section and add the appropriate subsection, probably ### Changed. During a release build, it automatically creates the version tag per the one used in the build and updates the file.